pkgs/by-name/no/noxdir/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "noxdir";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "crumbyte";
+    repo = pname;
```

do not reuse pname, write out repo names explicitly
```suggestion
    repo = "noxdir";
```